devel/qbe/Portfile

```
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                qbe
+version             1.0
+categories          devel
+platforms           darwin
+license             MIT
+maintainers         @alterae
```

Usually the maintainers section is written within curly braces, and `openmaintainer` appended:
```suggestion
maintainers         {@alterae openmaintainer}
```

`openmaintainer` means that other port maintainers are allowed to make minor updates to this port, unless there is a specific reason why it should be you and only you.